src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -70,6 +72,18 @@ private void InstrumentModule()
             }
         }
 
+        private bool IsExcludeFromCoverageAttribute(string attributeName)
+        {
+            var excludedAtrributeNames = new List<string>
+            {
+                "ExcludeFromCoverageAttribute",
+                "ExcludeFromCoverage",
+                nameof(ExcludeFromCodeCoverageAttribute),
+                nameof(ExcludeFromCodeCoverageAttribute).RemoveAttributeSuffix()
+            };
+            return excludedAtrributeNames.Contains(attributeName);
```

Comment:
Since the list of supported attributes is always going to be finite, why not just include the strings directly (`ExcludeFromCodeCoverage` and `ExcludeFromCodeCoverageAttribute`) and drop the suffix helper methods altogether

Reply:
Basically, I can included name of attributes as strings... Just after someone else's bad code with magic strings (which I had to deal with :) ), I always try to replace the lines with something more readable (nameof, type-safe enum, const, ...)

Reply:
Okay, I'll replace the strings directly